lib/pacta.js

```diff
@@ -282,7 +282,8 @@
          * resolve another promise with it. Return that promise as it is equivalent
          * to Promise b.
          */
-        this.map(f).map(function (pb) {
+        var mapped = this.map(f);
```

Comment:
Done, although now the comment seems disconnected from whats happening. But I don't want to take out that blank line. Looked too cramped. Is it fine with you as is?